src/nti/site/site.py

```diff
@@ -421,6 +441,7 @@ def __setstate__(self, state):
                 # Pure-python doesn't set _p_changed, but C does.
                 changed = reg._p_changed
                 reg.__class__ = BTreeLocalAdapterRegistry
+                reg._v_setting_state = False
```

Comment:
Failing to do this, leaves our converted registry thinking it is in the middle of ``__setstate__``